emojiHeatmapper/index.html

```diff
@@ -10,6 +10,18 @@
   <link rel="stylesheet" href="http://openlayers.org/en/v3.12.1/css/ol.css" type="text/css">
   <!-- OpenLayers JS -->
   <script src="http://openlayers.org/en/v3.12.1/build/ol.js" type="text/javascript"></script>
+  <script type="text/JavaScript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.6.4/jquery.min.js"></script>
+  <script type="text/JavaScript">
+    $(document).ready(function(){
+      var listItems= "";
+      $.getJSON("emoji.json", function(json) {
+      for (var i = 0; i < json.data.length; i++){
+        listItems+= "<option value='" + json.data[i][1] + "'>" + json.data[i][1] + "</option>";
```

Comment:
A small suggestion. Since there is already a file called script.js dedicated for application js logic, you may move this js code from html file to js file, keeping js and html separated helps in readability and maintaining code :)